### DISCUSSION TITLE
[RECIPIENT_ID] is not a valid Long

### DISCUSSION THREAD

**Discussion by ltcompounder** [plhery/node-twitter-api-v2#210] [Q&A]
Hey guys, having trouble sending a dm using the wrapper sendDM.

The recipient_id needs to be specified, so I specified my twitter handle. Error that I got back was '[RECIPIENT_ID] is not a valid Long'. I'm guessing twitter handle is not the same as recipient_id? If that's the case, how do I find my recipient_id?

**Comment by alkihis** [comment]
Your twitter handle, you mean your @?
You need to give your user ID, obtained through `.currentUser().id_str` for example.
